### PR TITLE
Fireeye hx fix

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py [hunk 1]
```diff
@@ -990,7 +990,7 @@ def file_acquisition_package_request(self, acquisition_id):
         response = self._http_request(
             method='GET',
             url_suffix=f'acqs/files/{acquisition_id}.zip',
-            headers=headers,
+            headers=self._headers | headers,
```
Comment (ilappe) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py:993]: please put a short comment regarding the need of both 
the new Accept and the token form the self._headers

**Review by ilappe — APPROVED** [review]
unit test can avoid the next bug here ..

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2_test.py [hunk 2]
```diff
@@ -1929,3 +1929,26 @@ def test_informative_error_in_get_token(mocker):
...
+
+
+def test_headers_file_acquisition_package_request(requests_mock, mocker):
+    """
+    Given:
+        - mock client, acquisition_id
+    When:
+        - running the file_acquisition_package_request
+    Then:
+        - ensure that the headers of this command is what expected.
```
Comment (ShahafBenYakir) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2_test.py:1941]: Let's mention the expected headers explicitly 
```suggestion
        - ensure that the headers of this command is _____
```

File: Packs/FireEyeHX/ReleaseNotes/2_2_10.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### FireEye Endpoint Security (HX) v2
+- Fixed an issue in **file_acquisition_command** where the command did not return the zip file.
```
Comment (ShahafBenYakir) [inline Packs/FireEyeHX/ReleaseNotes/2_2_10.md:4]: Right?
```suggestion
- Fixed an issue in **fireeye-hx-file-acquisition** where the command did not return the zip file.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**